config-templates/salesforce/readme.md

```diff
@@ -0,0 +1,15 @@
+## Overview
+This directory contains a set of config files for modeling salesforce data. It also includes several common metrics calcuated from salesforce leads and opportunities. If you see a metric that's missing please open an issue in this repo.
+
+The config files are based on tables loaded into your data warehouse using fivetran and use the dimensions, identifier and measures names from these tables. If you are using a diffrent ETL tool, you may need to update the dimension, identifier or measurers, but the structure of the table should be the same.
+
+## Steps to test
+
+1. Add thes configs to your metricflow model directory (this shoud be the same directory you specified in model_path during setup)
```

Comment:
sp: these
sp: should
nit: capitalize MetricFlow
nit: put model_path in code format `model_path`